### DISCUSSION TITLE
Plot rotation trace in IPDF

### DISCUSSION THREAD

**Discussion by hwhouse** [mtex-toolbox/mtex#570] [Ask Anything] [answered]
Dear community:

I got EBSD data of Copper sample before and after deformation, i want to demostrate the rotate of tensile axis (X axis in sample system) in crystal space using  IPDF,  now i can acquire the projection dot  of each grain in IPDF before and after deformation using the following command: plotIPDF(ori1, vector3d.X);
hold on;
plotIPDF(ori2, vector3d.X);
how can i plot a line to link the two dot to show the rotation trend like figure1.
![Figure1](https://user-images.githubusercontent.com/34276511/98762070-cbf7f100-2411-11eb-9492-fd73f0f913f9.PNG)

Best regards
Wang Hao

**Comment by kilir** [comment, marked as answer]
Hi,
have a look here: https://groups.google.com/g/mtexmail/c/a2X2vQw00yY/m/_WN1KwzWAgAJ
If you have a recent version of mtex (post 5.Sep.2020) if should be 
`plotIPDF([oriStart, oriStop], ref_direction ,'arrow' )` .
Cheers,
Rüdiger

**Reply by hwhouse** [reply]
Hi,
Thank your for the advice, it works, but for some grians, the two points are so close  that there is only arrow without  line jsut like this figure:
![PDF2_46_296](https://user-images.githubusercontent.com/34276511/98795518-43934380-2445-11eb-87cc-c165c6cd8add.png)
 Is there any method to change the shape and size of the arrow or just plot a line without arrow?

Best regards,
WangHao

**Reply by kilir** [reply]
It takes most options from Matlabs `arrow` so setting `'length'` and `'width'` would do what you want to do
Cheers,
Rüdiger

**Reply by hwhouse** [reply]
It really helps a lot， I really appreciate.

Best regards
Wang Hao
